### PR TITLE
Feat: cmd update rules

### PR DESCRIPTION
**Pull request by maxwelbm**
- command update rules engine

### CODE REVIEW

File: pkg/cmd/update/rules_engine/fixtures/missing.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "criteria": [
+      [
+        {
+          "operator": "starts_with",
+          "conditional": "if",
+          "input_value": "/"
+        }
+      ]
+    ]
+  }
```
Comment (helio-neto) [inline pkg/cmd/update/rules_engine/fixtures/missing.json:20]: Just a single line to be added at the file end.
```suggestion
  }
  
```

File: pkg/cmd/update/rules_engine/fixtures/response.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+              "conditional": "if",
+              "input_value": "/"
+            }
+          ]
+        ],
+      "is_active": true,
+      "order": 1
+    },
+    "schema_version": 3
+  }
```
Comment (helio-neto) [inline pkg/cmd/update/rules_engine/fixtures/response.json:26]: Maybe need to add a new line here too...
```suggestion
  }
  
```

File: pkg/cmd/update/rules_engine/fixtures/update.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      [
+        {
+          "variable": "${uri}",
+          "operator": "starts_with",
+          "conditional": "if",
+          "input_value": "/"
+        }
+      ]
+    ]
+  }
```
Comment (helio-neto) [inline pkg/cmd/update/rules_engine/fixtures/update.json:21]: And here too...
```suggestion
  }
  
```

**Review by helio-neto — DISMISSED** [review]
🚀

File: pkg/cmd/update/rules_engine/rules_engine.go [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+	cmd := &cobra.Command{
+		Use:           msg.Usage,
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+		$ azion update rules_engine"
+		$ azion update rules_engine -h"
+		$ azion update rules_engine --rule-id 1234 --application-id 1673635839 --phase request --in ruleengine.json"
```
Comment (PatrickMenoti) [inline pkg/cmd/update/rules_engine/rules_engine.go:58]: Two examples which are exactly the same

**Review by PatrickMenoti — DISMISSED** [review]

**Review by PatrickMenoti — APPROVED** [review]

**Review by helio-neto — APPROVED** [review]

**Pull request merged**